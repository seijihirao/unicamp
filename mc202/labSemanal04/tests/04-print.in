push 1    Ai que prazer - não cumprir um dever.
print-last
print-first
is-empty
push 2    Ter um livro para ler - e não o fazer!
print-last
print-first
is-empty
push 3    Ler é maçada, estudar é nada.
print-last
print-first
is-empty
push 4    O sol doira sem literatura. 
print-last
print-first
is-empty
push 5    O rio corre bem ou mal, sem edição original.
print-last
print-first
is-empty
push 6    E a brisa, essa, de tão naturalmente matinal, como tem tempo, não tem pressa...
print-last
print-first
is-empty
push 7    Livros são papéis pintados com tinta.
print-last
print-first
is-empty
push 8    Estudar é uma coisa em que está indistinta
print-last
print-first
is-empty
push 9    A distinção entre nada e coisa nenhuma.
print-last
print-first
is-empty
push 10   Quanto melhor é quando há bruma.
print-last
print-first
is-empty
exit
