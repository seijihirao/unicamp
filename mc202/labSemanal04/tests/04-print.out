1    Ai que prazer - não cumprir um dever.
1    Ai que prazer - não cumprir um dever.
nope
2    Ter um livro para ler - e não o fazer!
1    Ai que prazer - não cumprir um dever.
nope
3    Ler é maçada, estudar é nada.
1    Ai que prazer - não cumprir um dever.
nope
4    O sol doira sem literatura.
1    Ai que prazer - não cumprir um dever.
nope
5    O rio corre bem ou mal, sem edição original.
1    Ai que prazer - não cumprir um dever.
nope
6    E a brisa, essa, de tão naturalmente matinal, como tem tempo, não tem pressa...
1    Ai que prazer - não cumprir um dever.
nope
7    Livros são papéis pintados com tinta.
1    Ai que prazer - não cumprir um dever.
nope
8    Estudar é uma coisa em que está indistinta
1    Ai que prazer - não cumprir um dever.
nope
9    A distinção entre nada e coisa nenhuma.
1    Ai que prazer - não cumprir um dever.
nope
10   Quanto melhor é quando há bruma.
1    Ai que prazer - não cumprir um dever.
nope
